### PR TITLE
Add ESLint rule to prevent whole Lodash import

### PR DESCRIPTION
**Pull request by OliverJAsh**
Instead of importing all of Lodash, we must now be import individual modules, re. #11066.

This lint task will prevent us from accidentally importing all of Lodash, and thereby we can avoid file sizes jumping unexpectedly.

E.g.

``` js
// core.js
define(['lodash'], function () {});
```

Error:

```
$ grunt eslint:static/src
Running "eslint:static/src" (eslint) task

/Users/OliverJAsh/Development/frontend/static/src/javascripts/core.js
  1:9  error  Use Lodash modules instead  no-all-lodash-import
```

If you're using the ESLint plugin for Sublime, you have to add the rules to your global config, unfortunately: https://github.com/roadhump/SublimeLinter-eslint#i-want-to-use-custom-rules-global-eslintignore-file-etc

### CODE REVIEW

File: dev/eslint-rules/no-all-lodash-import.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+                lodashImports.forEach(function (literalNode) {
+                    context.report({
+                        node: literalNode,
+                        message: 'Use Lodash modules instead'
+                    });
+                });
+            }
+        }
+    };
+};
```
Comment (OliverJAsh) [inline dev/eslint-rules/no-all-lodash-import.js]: Here's the AST: http://astexplorer.net/#/DDNrKEEHYc

**Comment by sndrs** [comment]
:+1: great

**Comment by johnduffell** [comment]
do we need a test or two for this?  I assume you've manually tested but feels like a couple of unit tests could give more confidence

**Comment by OliverJAsh** [comment]
Good point John! See 9cf9256f38521bb080a2ad8730de1644939ad804

**Comment by johnduffell** [comment]
fantastic! now ur testing the thing that tests our code :confounded: :laughing: :+1:

**Pull request merged**